Resources/Maps/_Mono/Shuttles/vaquita.yml

```
@@ -0,0 +1,2118 @@
+  entities:
+  - uid: 88
+    components:
+    - type: Transform
+      rot: -1.5707963267948966 rad
+      pos: 4.5,4.5
+      parent: 1
+- proto: NFBedrollAlt
+  entities:
+  - uid: 109
+    components:
+    - type: Transform
+      pos: 3.4987547,3.5035484
+      parent: 1
+- proto: NFHolopadShip
+  entities:
+  - uid: 81
+    components:
+    - type: Transform
+      pos: 2.5,8.5
+      parent: 1
+- proto: NitrogenCanister
+  entities:
+  - uid: 111
+    components:
+    - type: Transform
+      anchored: True
+      pos: 1.5,2.5
+      parent: 1
+    - type: Physics
+      bodyType: Static
+- proto: OreBox
+  entities:
+  - uid: 82
+    components:
+    - type: Transform
+      pos: -2.5,2.5
+      parent: 1
+- proto: OreProcessor
+  entities:
+  - uid: 83
+    components:
+    - type: Transform
+      pos: -1.5,2.5
+      parent: 1
+- proto: OxygenCanister
+  entities:
+  - uid: 110
+    components:
+    - type: Transform
+      anchored: True
+      pos: 0.5,2.5
+      parent: 1
+    - type: Physics
+      bodyType: Static
+- proto: PortableGeneratorSuperPacmanShuttle
+  entities:
+  - uid: 79
+    components:
+    - type: Transform
+      pos: 3.5,5.5
+      parent: 1
+- proto: PoweredLEDSmallLight
+  entities:
+  - uid: 123
+    components:
+    - type: Transform
+      rot: -1.5707963267948966 rad
+      pos: 3.5,9.5
+      parent: 1
+- proto: Poweredlight
+  entities:
+  - uid: 122
+    components:
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: -3.5,7.5
+      parent: 1
+- proto: PoweredlightExterior
+  entities:
+  - uid: 114
+    components:
+    - type: Transform
+      rot: 3.141592653589793 rad
+      pos: -1.5,10.5
+      parent: 1
+  - uid: 115
+    components:
+    - type: Transform
+      rot: 3.141592653589793 rad
+      pos: 1.5,13.5
+      parent: 1
+- proto: PoweredWarmSmallLight
+  entities:
+  - uid: 121
+    components:
+    - type: Transform
+      pos: 3.5,5.5
+      parent: 1
+- proto: ReinforcedWindowDiagonal
+  entities:
+  - uid: 76
+    components:
+    - type: Transform
+      rot: 3.141592653589793 rad
+      pos: -1.5,6.5
+      parent: 1
+  - uid: 139
+    components:
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: 0.5,7.5
+      parent: 1
+  - uid: 207
+    components:
+    - type: Transform
+      rot: -1.5707963267948966 rad
+      pos: 0.5,8.5
+      parent: 1
+  - uid: 272
+    components:
+    - type: Transform
+      pos: -1.5,7.5
+      parent: 1
+- proto: SalvageTechfabNF
+  entities:
+  - uid: 68
+    components:
+    - type: Transform
+      pos: -1.5,4.5
+      parent: 1
+- proto: SignalButtonDirectional
+  entities:
+  - uid: 200
+    components:
+    - type: Transform
+      rot: -1.5707963267948966 rad
+      pos: -1.5,8.5
+      parent: 1
+    - type: DeviceLinkSource
+      linkedPorts:
+        218:
+        - - Pressed
+          - Toggle
+        217:
+        - - Pressed
+          - Toggle
+- proto: SMESAdvanced
```

looks good but i would do that.
```suggestion
- proto: SMESBasic
```